hbase_sql/src/main/java/com/splicemachine/derby/stream/spark/NativeSparkDataSet.java

```diff
@@ -1136,11 +1026,29 @@ public static boolean joinProjectsOnlyLeftTableColumns(JoinType joinType) {
     }
 
     @SuppressWarnings({ "unchecked", "rawtypes" })
-    public DataSet<ExecRow> writeAvroFile(DataSetProcessor dsp, int[] partitionBy, String location,
-                                          String compression, OperationContext context) throws StandardException {
+    public DataSet<ExecRow> writeAvroFile(DataSetProcessor dsp,
```

Comment:
note: this small function is added to remove a bigger Avro write function in SparkDataSet